src/runtime_src/core/edge/user/aie/graph.cpp

```diff
@@ -110,7 +110,7 @@ xclGMIOWait(xclDeviceHandle handle, const char *gmioName)
 
   if (!drv->isAieRegistered())
     throw xrt_core::error(-EINVAL, "No AIE presented");
-  auto aieArray = drv->getAieArray();
+  auto aieArray = drv->get_aie_array_shared();
```

Comment:
We are mixing Camel case and "_" seperated names in lot of places, Can you please fix all of them in next PR.
I know this is not from your change, but just to be consistent.

Reply:
sure, I could cleanup in next PR